### PR TITLE
add "vote button" to session page

### PR DESCRIPTION
**Pull request by anativ**

### CODE REVIEW

File: client/src/components/SessionPage.js [hunk 1]
```diff
@@ -57,14 +58,15 @@ const getDateAndTime = id => {
 };
 
 const dates = [
-  "October 15, 2017", // TODO change dates
-  "October 16, 2017"
+  "October 8, 2018", // TODO change dates
```
Comment (rantav) [inline client/src/components/SessionPage.js]: what? where did these hid? were they in use?...

Anyway I'd remove the `// TODO change dates` now

File: client/src/components/SessionPage.js [hunk 2]
```diff
@@ -57,14 +58,15 @@ const getDateAndTime = id => {
...
+  "October 9, 2018"
 ];
 
 
 const SessionPage = (props) => {
-  const { user, session, speakers, match: { params: { id } } } = props;
-  const {title, abstract, type, tags, outline, categories } = session;
+  console.log(props)
```
Comment (rantav) [inline client/src/components/SessionPage.js]: remove log

File: client/src/components/VoteButton.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import React, { Fragment } from 'react';
+import { Button } from 'reactstrap';
+import cn from 'classnames';
+import s from './SpeakerPage.css';
+
+export default ({ user, attended, proposalId, attendProposal }) => {
+  console.log({ user, attended, proposalId, attendProposal })
```
Comment (rantav) [inline client/src/components/VoteButton.js]: remove

File: client/src/components/SessionPage.js (hunk 2 above)
Comment (anativ) [reply client/src/components/SessionPage.js]: oops

File: client/src/components/SessionPage.js (hunk 1 above)
Comment (anativ) [reply client/src/components/SessionPage.js]: Note to self: We need to find all the configs in the code and put them in one place.... (not now)

**Comment by anativ** [comment]
@rantav All done, ready when you are...

**Pull request merged**